_includes/journals/volume1/issue1/paper1.html

```diff
@@ -0,0 +1,11 @@
+<h1>Paper Title</h1>
+
+<strong>Authors</strong>
+
```

Comment:
Please add additional information missing to satisfy crossref requirements including the abstract paragraph, DOI link, etc.